components/crossplane-control-plane/base/namespace-claim-configmap.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: namespace-claim-configmap
+  namespace: crossplane-control-plane
+data:
+  namespace-claim-cleaner.sh: |
+    #!/bin/bash
+
+    CURRENT_TIME=$(date +%s)
+    kubectl get namespaces.eaas.konflux-ci.dev -o json | jq -c '.items[]' | while read -r claim; do
+      CREATION_TIMESTAMP=$(echo $claim | jq -r '.metadata.creationTimestamp')
```

No pipeline required (untested but should work):
```suggestion
      CREATION_TIMESTAMP=$(jq -r '.metadata.creationTimestamp' <<< "$claim")
```